storage/innobase/dict/dict0load.cc

```diff
@@ -2298,8 +2298,8 @@ dict_load_tablespace(
 		table->file_unreadable = true;
 
 		if (!(ignore_err & DICT_ERR_IGNORE_RECOVER_LOCK)) {
-			sql_print_error("InnoDB: Failed to load tablespace "
-					ULINTPF " for table %s",
+			sql_print_error("InnoDB: Failed to load tablespace %"
+					PRIu32 " for table %s",
 					table->space_id, table->name);
```

Comment:
I think that we could at the same time correct the last argument to `table->name.str`, like it was done in #3493. In any case, when or after that change has been merged from 10.6 to 10.11, we will have to pay attention to this.

Reply:
Sure, if it helps with conflict resolution

Reply:
I suppose this thread is closed now that @grooverdan’ve merged the PR as is.